src/components/ResourceExplorer.tsx

```diff
@@ -43,32 +44,34 @@ const ResourceExplorer = ({ id, resource, history, parent }) => {
                 return (
                   <React.Fragment>
                     <Associator
-                      initialItems={associated[associatedType].resultSet}
                       editing={editing}
                       fetchItems={RESOURCE_MAP[associatedType].getList}
-                      fetchExitingAssociations={params =>
+                      fetchExistingAssociations={params =>
                         RESOURCE_MAP[associatedType].getList({
                           ...params,
                           [`${resource.name.singular}Id`]: id,
                         })
                       }
                       getName={RESOURCE_MAP[associatedType].getName}
+                      initialItems={associated[associatedType].resultSet}
                       onAdd={item => stageChange({ [associatedType]: { add: item } })}
                       onRemove={item => stageChange({ [associatedType]: { remove: item } })}
+                      type={associatedType}
                     />
                     {!parent &&
                       associated[associatedType].count >
-                        _.get(associated[associatedType], 'resultSet.length', 0) && (
+                        get(associated[associatedType], 'resultSet.length', 0) && (
                         <NavLink
                           to={`/${resource.name.plural}/${id}/${associatedType}`}
-                          style={{ fontSize: 14 }}
+                          style={{ color: LIGHT_BLUE, fontSize: 12 }}
```

Comment:
are these nav links totally unique or could we have a `smallLink` style that we reuse?

Reply:
Right now it looks like there are only a few uses of `NavLink` and they have different styles. I think if I end up reusing this style link as i go through Kim's redesign, I can create a reusable component.